### DISCUSSION TITLE
FarmPi and running OctoPrint instances

### DISCUSSION THREAD

**Discussion by mkevenaar** [mkevenaar/FarmPi#6] [General]
Please reply with the version of Raspberry Pi you are using (e.g. 3B+ / 4B-2GB) and the # of OctoPrint instances you are using.

```
Raspberry Pi:
OctoPrint instances: 
```

**Comment by calonmerc** [comment]
Raspberry Pi: 3B+
OctoPrint instances: 5

**Comment by chickencaptain** [comment]
Raspberry PI: 4 B-4gb
Octoprint Instances: 2

**Comment by Mistguard** [comment]
Raspberry Pi: 3B
OctoPrint instances: 2

**Comment by bratwiz** [comment]
Raspberry PI 4 B 4GB
Octoprint Instances 9
